tests_integration/zk_test_end_session.js

```diff
@@ -13,12 +13,12 @@ zk.connect(function (err) {
   if(err) throw err;
   assert.equal(zk.client_password.length, 32);
 
-  var zk2 = new ZK({connect: connect, timeout: timeout, debug_level: ZK.ZOO_LOG_LEVEL_INFO, host_order_deterministic: false,
+  var zk2 = new ZK({connect: connect, timeout: timeout, debug_level: constants.ZOO_LOG_LEVEL_INFO, host_order_deterministic: false,
              client_id: zk.client_id,client_password: zk.client_password});
   zk2.connect(function (err) {
     if(err) throw err;
     zk2.close();
-    zk.a_create ('/test_end_session', 'x', ZK.ZOO_SEQUENCE |ZK.ZOO_EPHEMERAL, function (rc, error, path)  {
+    zk.a_create ('/test_end_session', 'x', constants.ZOO_SEQUENCE | constants.ZOO_EPHEMERAL, function (rc, error, path)  {
       console.log ("zk node create result: %d, error: '%s', path=%s", rc, error, path);
       assert(rc != 0);
```

Comment:
This assertion fails - the rc in this case is `0`. Is it ok?

Reply:
Ok! I can reproduce it in the Master branch. It happens occasionally. Long term, I am planning to more rely on integration tests that use `tape` and assertions, rather than the existing ones (written a long time ago).